code/_expts/slotnick_etal_2001/analyze/stats/de_StaticizerCC.m

```diff
@@ -3,93 +3,76 @@
-            end
-        end
+        trial_types = {'easy', 'hard'};
         type1_perf = tmp(:, guru_instr(mSets.data.(ds).TLAB, trial_types{1})); %2 freqs
         type2_perf = tmp(:, guru_instr(mSets.data.(ds).TLAB, trial_types{2}));
         stats.perf.(ds){mi} = {type1_perf type2_perf};
-        
     end;
-    
-    
+
+
     % Now do some reporting
     fprintf('\n');
     for mi=1:length(mss)
-        fprintf('[%s] Sig=%5.2f:\tOn: %5.2e +/- %5.2e\tOff: %5.2e +/- %5.2e\n', ds, mss{mi}(1).sigma, ...
-            mean(stats.perf.(ds){mi}{1}(:)), std (stats.perf.(ds){mi}{1}(:)), ...
-            mean(stats.perf.(ds){mi}{2}(:)), std (stats.perf.(ds){mi}{2}(:)) ...
-            );
+        fprintf('[%s] Sig=%5.2f:\t%s: %5.2e +/- %5.2e\t%s: %5.2e +/- %5.2e\n', ...
+            ds, mss{mi}(1).sigma, ...
+            trial_types{1}, ...
+            mean(stats.perf.(ds){mi}{1}(:)), std(stats.perf.(ds){mi}{1}(:)), ...
+            trial_types{2}, ...
+            mean(stats.perf.(ds){mi}{2}(:)), std(stats.perf.(ds){mi}{2}(:)) ...
+        );
     end;
-    
-    
+
+
     %%%%%%%%%%%%%%%%%%
     % Now test for significance
     %%%%%%%%%%%%%%%%%%
-    
+
     % Can't do stats with a single sigma
     if (length(stats.perf.(ds)) < 2),  return; end;
-    
+
     % Use the first and last sigmas as the two to compare
     perf = stats.perf.(ds)([1 end]);
-    
-    
-    
+
+
+
     % First, test for main interaction of on/near vs. off/far
-    
+
     % Vector of all dependent measures:
     % cell 1 is on/near responses, 2 is off/far; average over all instances of each stim class
     Y = [ mean(perf{1}{1},2); mean(perf{2}{1},2); ...
           mean(perf{1}{2},2); mean(perf{2}{2},2) ];
-    
+
     % Subject names: "RH-and-model instance number" and "LH-and-model-instance-number"
     S = horzcat( guru_csprintf('RH%d', num2cell(1:size(perf{1}{1},1))), ...
         guru_csprintf('LH%d', num2cell(1:size(perf{2}{1},1))), ...
         guru_csprintf('RH%d', num2cell(1:size(perf{1}{2},1))), ...
         guru_csprintf('LH%d', num2cell(1:size(perf{2}{2},1))) ...
         )';
-    
+
     % Factor 1: Hemisphere
     F1 = [ repmat({'RH'},   [size(perf{1}{1},1) 1]); repmat({'LH'},   [size(perf{2}{1},1) 1]); ...
          repmat({'RH'},   [size(perf{1}{2},1) 1]); repmat({'LH'},   [size(perf{2}{2},1) 1]) ];
-    
+
     % Factor 2: On/off or Near/far
     F2 = [ repmat({trial_types{1}}, [size(perf{1}{1},1) 1]); repmat({trial_types{1}}, [size(perf{2}{1},1) 1]); ...
-         repmat({trial_types{2}}, [size(perf{1}{2},1) 1]); repmat({trial_types{2}}, [size(perf{2}{2},1) 1]) ];
-    
+           repmat({trial_types{2}}, [size(perf{1}{2},1) 1]); repmat({trial_types{2}}, [size(perf{2}{2},1) 1]) ];
+
     % Convert all above labels to numeric
     [~,~,S_n] = unique(S);
     F1_n = zeros(size(F1)); F1_n(strcmp(F1,'RH')) = 1; F1_n(strcmp(F1,'LH')) = 2;
     F2_n = zeros(size(F2)); F2_n(strcmp(F2,trial_types{1})) = 1; F2_n(strcmp(F2,trial_types{2})) = 2;
-    
-    
```

Comment:
@vishaalprasad Note all the trailing whitespace in your code. You should install a git hook that removes trailing whitespace, looks for merge conflict markers, and other things.  E.g. http://makandracards.com/makandra/11541-how-to-not-leave-trailing-whitespace-using-your-editor-or-git